src/main/java/cc/hyperium/mods/chromahud/ElementRenderer.java

```diff
@@ -292,8 +294,7 @@ public void renderElements() {
             try {
                 element.draw();
             } catch (Exception ignored) {
-                // lmao this is so unsafe sk1er XDDDDDDDDD
-                // Dude it's in a try catch so that it IS safe!
+                // 1000% safe ok?
```

Comment:
Could we just remove these informal comments and either leave it blank or put in a TODO to improve it.